doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -487,6 +487,10 @@ Other libraries
   * Changed the library implementation to bypass the flash driver when storing the emergency data.
     This allows calling the :c:func:`emds_store` function from an interrupt context.
 
+* :ref:`mod_dm`:
+  * Added run-time ranging window length configuration.
+  * Improved the calculation of the mpsl timeslot length by using the functionality of the :ref:`nrf_dm` library.
```

Comment:
```suggestion
  * Added window length configuration for the runtime ranging window.
  * Improved the calculation of MPSL timeslot length by using the :ref:`nrf_dm` library functionality.
```

Reply:
I think the point with original sentence is that the "ranging window" is configurable at runtime now, there is no "runtime ranging window" :nerd_face:

Reply:
Maybe something like this:
_Added window length configuration in run-time, when a new measurement request is added._

Reply:
Right, my bad. I just had an issue with the long list of nouns, not really understanding what defines what.

Would this be clear: "Added a window length configuration to be used runtime, when a new measurement request is added."?